src/lib/clients/ansible.js

```diff
@@ -20,6 +20,14 @@ class Ansible {
   }
 
   async sync() {
+    const util = require('util');
+    const exec = util.promisify(require('child_process').exec);
+
+    async function ls() {
+      const { stdout, stderr } = await exec('ansible-galaxy collection install --force -p ../../../ansible w3f.polkadot_validator:0.0.3');
```

Comment:
The version of the collection needs to be defined as a configuration value, having it hardcoded makes things much harder to manage.

Reply:
executed here:
https://github.com/w3f/polkadot-secure-validator/blob/cc0d47b43e8b4aa063db4870286518aaf4594baa/src/lib/clients/ansible.js#L24-L26

value here:
https://github.com/w3f/polkadot-secure-validator/blob/cc0d47b43e8b4aa063db4870286518aaf4594baa/config/main.sample.json#L15
wdyt?